### ISSUE TITLE
Look at matched clusters in RM vs. WaZP -- for the clusters with significantly different redshift, why?

### ISSUE THREAD

**Issue by chihway** [ajamsellem/WaZP-RedMaPPer_comparison#3]
Are these primarily low S/N clusters? or at some particular redshift ranges?

**Comment by ajamsellem** [comment]
![redshift_difference_v_radius](https://user-images.githubusercontent.com/37815753/53903388-33d52200-4009-11e9-954f-9b1ebe87e84b.png)

**Comment by ajamsellem** [comment]
![redshift_difference_v_radius](https://user-images.githubusercontent.com/37815753/53903641-cfff2900-4009-11e9-83a0-b8205505aa4e.png)

The higher differences in redshift seem to be evenly distributed over radius.

**Comment by ajamsellem** [comment]
![large_diff_redshift_compare](https://user-images.githubusercontent.com/37815753/53903709-fcb34080-4009-11e9-9f24-84633481afb4.png)

The higher differences in redshift seem to be evenly distributed over sky location.

**Comment by chihway** [comment]
Nice plots!

For RM, it seems like the smaller clusters are more prone to large z differences, but not sure why the radius distributions are so different between the two catalogs. You can compare the histogram of the radiuses for the two matched catalogs, and also a scatter plot of R_RM vs. R_WaZP. As you said, it could just be that they are different quantities.  

"The higher differences in redshift do not seem to be evenly distributed over sky location." --> hmm, they seem pretty random to me, how did you make this observation?

**Comment by ajamsellem** [comment]
> Nice plots!
> 
> For RM, it seems like the smaller clusters are more prone to large z differences, but not sure why the radius distributions are so different between the two catalogs. You can compare the histogram of the radiuses for the two matched catalogs, and also a scatter plot of R_RM vs. R_WaZP. As you said, it could just be that they are different quantities.
> 
> "The higher differences in redshift do not seem to be evenly distributed over sky location." --> hmm, they seem pretty random to me, how did you make this observation?

I'll do some more plots on the radii measurements of the two data sets to see if I can find an explanation. Although we should note that the radius axis has a range ~0.5-1.0 Mpc, while WaZP is from ~0.2-1.0. The radii where they overlap, however, seem to be in agreement.

That statement about redshift distribution was typo. It should have said the exact opposite, but I've corrected it now.

**Comment by ajamsellem** [comment]
![image](https://user-images.githubusercontent.com/37815753/54088997-ee2e9700-4331-11e9-899a-3447581f6118.png)

Histogram showing the distribution of radii for the 2 data sets. Note that both data sets had a few cluster radii > 1.0 Mpc, but since there are so few of them (~50 for RM and ~10 for WaZP), they cannot be seen on the histogram (even if we had a longer x-axis). Similarly, there are ~15 WaZP radii below 0.2 Mpc.

**Comment by ajamsellem** [comment]
![image](https://user-images.githubusercontent.com/37815753/54300123-38aa5080-458a-11e9-9260-8cb43bc70a47.png)

This plot seems quite illustrative. First it seems as though the biggest discrepancies appear when there is a larger redshift in at least one of the data sets. The good news is that it does not seem as though one data set is systematically over- or under-estimating the redshift as the discrepancies are pretty evenly distributed about the line where we'd expect the clusters to lie (i.e. where the cluster redshifts from both data sets would be in agreement). One may argue that RM redshifts tend to be slightly higher than WaZP redshifts in the higher redshift regime and that WaZP redshifts tend to be slightly higher than RM redshifts in the lower redshift regime, but only slightly. Finally, the one large discrepancy (of about 0.35) is very clearly an anomaly.

**Comment by chihway** [comment]
This plot is nice indeed. Just to be clear, the "gap" around the line is by construction, right? it's your definition of a large difference? 

I think there is some structure in this data that's kind of strange -- say, if you look at RM at ~0.6 (0.9), the WaZP redshifts will be systematically high (low), whereas if you look at some fixed WaZP redshift the RM redshifts are pretty unbiased.

**Comment by ajamsellem** [comment]
Yes, the gap is by construction. If we included all data points, I would expect to get plenty of points more clustered on the line.

It does seem as though we have larger discrepancies around redshifts of 0.6 and 0.9. The issue could be that as we get closer to the max redshift of each data set (~0.75 for WaZP and ~0.9 for RM), the redshifts become less reliable? That's the only commonality between these two regions that I see.
